### ISSUE TITLE
GraphQL analysis regression

### ISSUE THREAD

**Issue by finestructure** [SwiftPackageIndex/SwiftPackageIndex-Server#738]
Before:

<img width="1240" alt="Screenshot 2020-10-05 at 12 05 33" src="https://user-images.githubusercontent.com/65520/95078569-a3475200-0715-11eb-9b16-4676dc8d333a.png">

After:

<img width="1166" alt="Screenshot 2020-10-05 at 14 13 37" src="https://user-images.githubusercontent.com/65520/95078588-aa6e6000-0715-11eb-98bc-dd11da0712fe.png">

- release count off (and the same) in both cases (but I believe this is because we don't count certain tags - ignoring this for now)
- last issue closed is the same in both cases and "1 day ago" may actually be correct. It's hard to confirm manually, because the closure can be of any of the back pages. The later close date is typically correct here, so this looks fine.
- last pull request closed is definitely a regression:

<img width="478" alt="Screenshot 2020-10-05 at 14 22 23" src="https://user-images.githubusercontent.com/65520/95079004-4ef0a200-0716-11eb-9342-adf953d087af.png">

Sorting by "last updated" shows why this might go wrong. The assumption was that merged PRs aren't updated post merge but that's not correct:

<img width="481" alt="Screenshot 2020-10-05 at 14 22 39" src="https://user-images.githubusercontent.com/65520/95079074-6b8cda00-0716-11eb-9242-e0942c51a786.png">

It's still weird that it'd be "2 months ago", because if we take the last entry it should be "2 years ago".

What might happen is that it picks up the first closed PR from Aug 13 (i.e. ~ 2 months):

<img width="452" alt="Screenshot 2020-10-05 at 14 25 23" src="https://user-images.githubusercontent.com/65520/95079233-9d05a580-0716-11eb-927e-0213b6515c18.png">

Pretty sure that's it.

**Comment by finestructure** [comment]
Confirmed, it's picking up the last closed over the last merged:

```
      "closedPullRequests": {
        "nodes": [
          {
            "closedAt": "2020-08-13T19:10:08Z"
          }
        ]
      },
     ...
      "mergedPullRequests": {
        "nodes": [
          {
            "closedAt": "2017-10-30T01:43:44Z"
          }
        ]
      },
```

In both cases (like for issues) we need to pull down a list and filter client side.

**Comment by finestructure** [comment]
Fixed in staging, rolling out as 2.2.0 (bumping minor due to significant change)

<img width="1206" alt="Screenshot 2020-10-06 at 09 53 33" src="https://user-images.githubusercontent.com/65520/95174955-0a6b1200-07bb-11eb-8b49-38b210e813d1.png">
